source/_integrations/tesla_fleet.markdown

```diff
@@ -64,7 +64,14 @@ Tesla restricts open-source integrations to the ["Discovery" plan](https://devel
 
 ## Command signing
 
-Certain vehicles, including all vehicles manufactured since late 2023, require vehicle commands to be encrypted end-to-end and signed with a private key. The Tesla Fleet integration is unable to perform this encryption at this time, so certain features may be disabled or throw an exception when used.
+Certain vehicles, including all vehicles manufactured since late 2023, require vehicle commands to be signed with a private key. You will need to use Tesla's [command line tools](https://github.com/teslamotors/vehicle-command/blob/main/README.md#installation-and-configuration) to generate a key pair and install the public key on your vehicle using Bluetooth.
```

Comment:
Somehow here it should be mentioned what happens if the user does not perform these steps e.g. the vehicle requires signing but it's not performed.
Can it somehow be clarified what remains of the integrations in that case.

Reply:
@c0ffeeca7 any help from your side. I think the last change was pretty good but maybe it can be clarified/highlighted better?
I think point 1 and 2 are given, just number 3 needs to be highlighted so the user knows the limitations of not providing this key.